packages/opal-server/opal_server/config.py

```diff
@@ -99,7 +99,8 @@ class OpalServerConfig(Confi):
         False,
         "Set if OPAL server should use a fixed clone path (and reuse if it already exists) instead of randomizing its suffix on each run",
     )
-    POLICY_REPO_MAIN_BRANCH = confi.str("POLICY_REPO_MAIN_BRANCH", "master")
+    POLICY_REPO_MAIN_BRANCH = confi.str("POLICY_REPO_MAIN_BRANCH", None)
+    POLICY_REPO_TAG = confi.str("POLICY_REPO_TAG", None)
```

Comment:
This is great, but notice that this only works with OPAL working with a single repo: branch/tag -  and won't enable this feature for the OPAL scopes option. 
If you're feeling up to it, it might be worth [adding there](https://github.com/permitio/opal/blob/master/packages/opal-common/opal_common/schemas/policy_source.py#L54) as well

Reply:
Right - I'll add it there as well. I've never used OPAL scopes myself so I'll need to read up a bit on how to test it.